src/bash/unittest/plugin_test.c

```diff
@@ -0,0 +1,216 @@
+#include <stdio.h>
+#include <string.h>
+#include <CUnit/CUnit.h>
+#include <CUnit/Basic.h>
```

Comment:
>CUnit [](http://example.com/codeflow?start=10&length=5)

Does bash has some test framework? We are using gtest in other components.

Reply:
Bash not have source code level UT, it's only have some bash script UT to test built-in commands, all those test case are under 'tests' folder.